planet/src/main/java/com/google/planet/servlets/SendEmailServlet.java

```diff
@@ -34,47 +35,60 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import javax.servlet.annotation.WebServlet;
+import java.io.BufferedReader;
+import java.util.stream.Collectors;
 
 @WebServlet("/send-itinerary-to-email")
 public class SendEmailServlet extends HttpServlet {
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        Properties props = new Properties();
-        Session session = Session.getDefaultInstance(props, null);
-        try {
-            Message message = new MimeMessage(session);
-            message.setFrom(new InternetAddress("alicexyz@google.com", "Alice Zhou"));
-            //TODO: change this to the signed in user's email address
-            message.addRecipient(Message.RecipientType.TO,
-                            new InternetAddress("alicexyz@google.com"));
-            //TODO: add custom subject, html body, and attached file
-            message.setSubject("Your itinerary for Toronto trip");
-            Multipart multipartContent = new MimeMultipart();
+        BufferedReader br = request.getReader();
+        String userEmail = request.getParameterMap().get("email")[0];
+        String listName = request.getParameterMap().get("listName")[0];
+        String itineraryContent = "";
 
-            // Add email body
-            MimeBodyPart htmlPart = new MimeBodyPart();
-            String htmlBody = "<div>Hello! <h3> Thank you for using Planet.</h3> " +
-                "<h3>You can see a copy of your itinerary in the attachment below.</h3></div>";  
-            htmlPart.setContent(htmlBody, "text/html");
-            multipartContent.addBodyPart(htmlPart);
+        if (br != null) {
+            itineraryContent =  br.lines().collect(Collectors.joining("\n"));
+            Properties props = new Properties();
+            Session session = Session.getDefaultInstance(props, null);
+            
+            try {
+                Message message = new MimeMessage(session);
+                message.setFrom(new InternetAddress("alicexyz@google.com", "Alice Zhou"));
```

Comment:
nit: add a TODO to set up an email for Planet? :P

Reply:
Okay!